src/command_modules/azure-cli-sql/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 
 2.0.20
 ++++++
+* Added az sql db rename
 * Clean up unused test files
```

Comment:
Consider removing this since it's not a user facing change.

Reply:
Can do, but it was Troy who added this :)

Reply:
To be clear, you mean the "Clean up unused test files" since the new command definitely is customer-facing

Reply:
Yes I was referring to "Clean up unused test files".
@troydai I don't think this should have been added in the first place.